### ISSUE TITLE
Revert FAQ and Issues changes once Map interface stabilizes

### ISSUE THREAD

**Issue by dcwalk** [ixmaps/website2017#35]
Demo launch changes were made to reflect the fact that the new map interface hadn't landed. 
See #29 and #19 for those changes.

These need to be changed once the Map is stabilized

**Comment by colinmccann** [comment]
Yeah, my bad - I had thought that I had done this, but definitely missed a couple. Will go through closed issues and try to add

**Comment by dcwalk** [comment]
I'd like to keep this open until the changes have been reverted if possible.

**Comment by dcwalk** [comment]
A chunk of Learn Issues page changes have been reverted/updated here: https://github.com/ixmaps/website2017/commit/2b58c39c4ea3da10a5c24e7b1c8e357700e4c39c

I'd like to remove all the commented out sections once we launch
